api/tools/src/main/java/org/pmiops/workbench/tools/cdrconfig/UpdateCdrConfig.java

```diff
@@ -86,14 +83,9 @@ public CommandLineRunner run(
    *   <li>duplicate IDs (or lack one)
    *   <li>have an archived default version
    *   <li>belong to a tier which is not also present in this file
-   *   <li>have more than one default version per tier
```

Comment:
most of this javadoc should be restored - notice that the exceptions are when allowEmptyTiers is true, which is no longer possible.

Reply:
Good catch. Thanks for pointing this out.